src/styles/constants/_animation.scss

```diff
@@ -1 +1 @@
-$bdl-transition-duration-200ms: .2s;
+$bdl-transition-duration-base: .15s;
```

Comment:
Since we are renaming this, might as well convert to camelcase SUIT style, eg: is in the CSS conventions doc

Reply:
`$bdl-transitionDurationBase`?